casper/src/test/scala/coop/rchain/casper/rholang/RuntimeSpec.scala

```diff
@@ -9,16 +9,17 @@ import coop.rchain.rholang.interpreter.accounting.Cost
 import coop.rchain.rspace.hashing.Blake2b256Hash
 import coop.rchain.rspace.syntax.rspaceSyntaxKeyValueStoreManager
 import coop.rchain.shared.Log
-import coop.rchain.shared.scalatestcontrib.effectTest
 import coop.rchain.store.InMemoryStoreManager
 import monix.eval.Task
-import org.scalatest.flatspec.AnyFlatSpec
+import monix.execution.Scheduler
+import monix.testing.scalatest.MonixTaskTest
+import org.scalatest.flatspec.AsyncFlatSpec
 import org.scalatest.matchers.should.Matchers
 
-class RuntimeSpec extends AnyFlatSpec with Matchers {
-  import monix.execution.Scheduler.Implicits.global
+class RuntimeSpec extends AsyncFlatSpec with MonixTaskTest with Matchers {
+  implicit override def scheduler: Scheduler = Scheduler.io("monix-task-support-spec")
```

Comment:
Let's not change existing scheduler until we decide what would be the best. If not specified _global_ will be used.

Reply:
[Done](https://github.com/rchain/rchain/pull/3727/commits/50c1493066666f352050cb2c8060ecf13fb93b4c).